examples/bank/bank.go

```diff
@@ -187,11 +145,11 @@ func (bank *Bank) continuousMoneyTransfer() {
 
 // Initialize all the bank accounts with cash.
 func (bank *Bank) initBankAccounts(cash int64) {
-	calls := make([]client.Call, bank.numAccounts)
+	batch := &client.Batch{}
 	for i := 0; i < bank.numAccounts; i++ {
-		calls[i] = client.Put(makeAccountID(i), []byte(fmt.Sprintf("%d", cash)))
+		batch = batch.Put(makeAccountID(i), fmt.Sprintf("%d", cash))
```

Comment:
Maybe you should add an `account` type for holding the account balance. I think it would actually make the encoding and decoding cleaner as it would be localized next to the account struct:

```
type account struct {
  Balance int64
}

func (a account) Encode() ([]byte, error) {
  return json.Marshal(a)
}

func (a *account) Decode(b []byte) error {
  return json.Unmarshal(b, a)
}
```